README.md

```diff
@@ -145,6 +156,174 @@ readData(dataInput) {
+
+
+# Securing Client Connection
+
+## 1. Supported Security Protocols
+
+This chapter describes the security features of Hazelcast Node.js Client. These features allow you to perform security activities, such as TLS/SSL and its sub-section Mutual Authentication. All of the Security features explained in this chapter are the features of **Hazelcast IMDG Enterprise** edition.
+
+### 1. TLS/SSL
+
+One of the offers of Hazelcast is the TLS/SSL protocol which you can use to establish an encrypted communication across your cluster with key stores and trust stores.
+
+- A Java `keyStore` is a file that includes a private key and a public certificate. The equivalent of a key store is the combination of `key.pem` and `cert.pem` files with pem extension at the Node.js client side.
+
+- A Java `trustStore` is a file that includes a list of certificates trusted by your application which is named certificate authority. The equivalent of a trust store is a `ca.pem` file with pem extension at the Node.js client side.
+
+#### 1.1. TLS/SSL for Hazelcast Members
+
+Hazelcast allows you to encrypt socket level communication between Hazelcast members and between Hazelcast clients and members, for end to end encryption. To use it, see [TLS/SSL for Hazelcast Members section](http://docs.hazelcast.org/docs/3.10.4/manual/html-single/index.html#tls-ssl-for-hazelcast-members).
+
+#### 1.2. TLS/SSL for Hazelcast Node.js Clients
+
+Hazelcast Node.js clients which support TLS/SSL should have the following a user supplied SSL `options` object to pass to
+[`tls.connect` of Node.js](https://nodejs.org/api/tls.html#tls_tls_connect_options_callback):
+
+```javascript
+var fs = require('fs');
+
+var clientConfig = new Config.ClientConfig();
+clientConfig.networkConfig.sslOptions = {
+    rejectUnauthorized: true,
+    ca: [fs.readFileSync(__dirname + '/server-cert.pem')],
+    servername: 'foo.bar.com'
+};
+```
+
+#### 1.3. Mutual Authentication
+
+As explained above, Hazelcast members have key stores used to identify themselves (to other members) and Hazelcast clients have trust stores used to define which members they can trust.
+
+Starting with Hazelcast 3.8.1, mutual authentication is introduced. This allows the clients also to have their key stores and members to have their trust stores so that the members can know which clients they can trust.
+
+To enable mutual authentication, you need to set the following property at the Java member side:
+
+```java
+props.setProperty("javax.net.ssl.mutualAuthentication", "REQUIRED");
```

Comment:
keep in mind that the audience should not be writing java code. They are using standalone servers and config file. Give xml configuration instead.